Source/ACE.Entity/AceObjectPropertiesAttribute.cs

```diff
@@ -5,23 +5,57 @@
 namespace ACE.Entity
 {
     [DbTable("ace_object_properties_attribute")]
-    public class AceObjectPropertiesAttribute : ICloneable
+    public class AceObjectPropertiesAttribute : BaseAceProperty, ICloneable
     {
-        [DbField("aceObjectId", (int)MySqlDbType.UInt32, IsCriteria = true, ListGet = true, ListDelete = true)]
-        public uint AceObjectId { get; set; }
-
-        [DbField("attributeId", (int)MySqlDbType.UInt16)]
+        private uint _xpSpent = 0;
+        private ushort _ranks = 0;
+        private ushort _base = 0;
+        
+        [DbField("attributeId", (int)MySqlDbType.UInt16, IsCriteria = true, Update = false)]
         public ushort AttributeId { get; set; }
 
         [DbField("attributeBase", (int)MySqlDbType.UInt16)]
-        public ushort AttributeBase { get; set; }
+        public ushort AttributeBase
```

Comment:
Its moments like this I miss C++ templates and C macros.  All these manual IsDirty copies...